contract.md

```diff
@@ -12,29 +19,52 @@ like to work on, as part of your project (for example, other things related to D
 you have come across, or topics from other classes)? Why are you interested in working on
 these things?_
 
+A skill that we covered in this class that I would like to work on is implementing a DSL externally using packrat parsers. This was a tool that I found really cool when we went over it in class but I didn't choose to use it on the Picobot assignment. Since I struggled when trying to implement an internal DSL for the project, I would like to take a stab at writing an external DSL for this one and potentially use packrat parsing. Also, I currently think that my project will be better implemented as an external DSL which provided additional motivation for me to learn how to use this skill.
+
+Something outside of this class that I would like the chance to work on would be to get the application side of my project up and running. So far through the HMC CS curriculum, I haven't gained very much experience writing projects that have a polished front-end result that the user can interact with. This is something that I have been interested in learning how to create but haven't gained any experience learning inside the classroom. So, I am really looking forward to connecting my DSLs to a front-end application that I can create and design. 
+
```

Comment:
I agree that your project seems more suitable as an external DSL given the flexibility that is needed to design the syntax. One thing to consider is that the 2nd DSL may be more suitable as an internal DSL depending on the output of the 1st DSL. 
It would be super cool to have a tool wrapped around this DSL!